src/planner/drop_plan.cpp

```diff
@@ -40,16 +40,22 @@ DropPlan::DropPlan(parser::DropStatement *parse_tree) {
       break;
     }
     case parser::DropStatement::EntityType::kTrigger: {
-      // note parse_tree->table_name is different from parse_tree->GetTableName()
+      // note parse_tree->table_name is different from
+      // parse_tree->GetTableName()
       database_name = parse_tree->GetDatabaseName();
       table_name = std::string(parse_tree->table_name_of_trigger);
       trigger_name = std::string(parse_tree->trigger_name);
       drop_type = DropType::TRIGGER;
       break;
     }
-    default: {
-      LOG_ERROR("Not supported Drop type");
+    case parser::DropStatement::EntityType::kIndex: {
+      database_name = parse_tree->GetDatabaseName();
+      table_name = std::string(parse_tree->table_name_of_trigger);
```

Comment:
You are not using table_name or database_name in the executor. Delete them.

Actually, I am not sure if database_name is useful. In Postgres, index name is not visible across databases, but our catalog does not expose the database name in the API. @mengranwo Can you explain how the index is implemented in Peloton catalog?